battan40/Tainted_Tomatillos
# Feature/final refactor functionality

### Is this a fix or a feature?
- This is a fixture

### What is the change or fix?
- It adjusts some of our css styles as they function on the page for better responsive design
- Cleans our test file structures for more streamlined to the two views available
- Clarifies/Refactors tests for true green passes and sad path testing
- Final sweep for organization/lint/spacing/lighthouse/methods outside the render/cleaning inside the utils

### How should this be tested?
- Please shrink and expand the screen to verify page is responsive
- Please deploy to url listed below
- Open and verify that search button is functional
- Back button is functional
- And reload is the true clearer of the main page view

### Linked Issues: 
- closes #36 #22 #11 #15 #35
